libtextsecure/message_receiver.js

```diff
@@ -154,7 +155,16 @@ MessageReceiver.prototype.extend({
     };
     this.httpPollingResource.handleMessage(message, options);
   },
-  handleUnencryptedMessage({ message }) {
+  async handleUnencryptedMessage({ message }) {
+    const conversation = await window.ConversationController.getOrCreateAndWait(
```

Comment:
Move the conversation get inside the if below?

Reply:
updated